homeassistant/components/media_player/dlna_dmr.py

```diff
@@ -124,7 +124,7 @@ def wrapper(self, *args, **kwargs):
 
     # build upnp/aiohttp requester
     from async_upnp_client.aiohttp import AiohttpSessionRequester
-    session = async_get_clientsession(hass)
+    session = async_get_clientsession(hass, verify_ssl=False)
```

Comment:
why is this? That doesn't seem right.

Reply:
In another issue there were some people having problems with upnp connections. It turned out all those people had SSL enabled. As an attempt to help them, not really knowing where to look, this was an attempt. Seems I left it there.